resources/pt/cemu.po

```diff
@@ -1752,8 +1945,8 @@ msgid "&H264 API"
 msgstr "&H264 API"
 
 #: MainWindow.cpp
-msgid "&NFP API"
-msgstr "API &NFP"
+msgid "&Graphic pack patches"
+msgstr "Correções de &pacotes gráficos"
```

Comment:
In this string "patches" refers to the ability of Cemu to apply patches (mods) to games through graphic packs. It seems to me that you're translating "patches" as "corrections", though I may be wrong.